WordPress/src/main/res/layout/theme_activation_bottom_sheet.xml

```diff
@@ -0,0 +1,169 @@
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/use_theme_layout_ll"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_medium">
+
+        <ImageView
+            android:id="@+id/keep_current_check_iv"
+            android:layout_width="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/theme_activation_bottom_sheet_selected_description"
+            android:src="@drawable/ic_tick_vector_blue" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            style="@style/ThemeActionSheetOptionText"
+            android:layout_height="@dimen/theme_action_sheet_option_height"
+            android:layout_alignParentStart="true"
+            android:layout_toStartOf="@id/keep_current_check_iv"
+            android:gravity="center_vertical"
+            android:text="@string/theme_activation_keep_current_layout_main_text" />
+    </RelativeLayout>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="70dp"
+        android:layout_below="@id/keep_current_layout_ll"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:background="@color/theme_action_sheet_button_layout_bg">
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_divider_height"
+            android:background="@color/gray_5" />
+
+        <LinearLayout
+            android:id="@+id/action_sheet_buttons_ll"
+            android:layout_width="match_parent"
+            android:layout_height="55dp"
+            android:layout_marginBottom="@dimen/margin_medium"
+            android:layout_marginEnd="@dimen/margin_extra_large"
+            android:layout_marginStart="@dimen/margin_extra_large"
+            android:layout_marginTop="@dimen/margin_medium"
+            android:orientation="horizontal">
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/preview_theme_button"
+                style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                android:text="@string/theme_activation_bottom_sheet_preview_button_label"
+                android:textAllCaps="false"
+                android:textColor="@color/theme_action_sheet_preview_button"
+                app:strokeColor="@color/theme_action_sheet_preview_button"
+                app:cornerRadius="8dp" />
```

Comment:
This [approach suggested above](https://github.com/wordpress-mobile/WordPress-Android/pull/15029/files#r671234018) can be used here too.  In this case I think it might work better if instead we created a style for the button as it is done [here](https://github.com/wordpress-mobile/WordPress-Android/blob/9cbbea8d82610a60f32a4c4fce5ec467a5275873/WordPress/src/main/res/values/styles.xml#L1424). Wdyt?